interfaces/apparmor/backend_test.go

```diff
@@ -601,6 +601,41 @@ func (s *backendSuite) TestCombineSnippetsOpenSUSETumbleweedOldKernel(c *C) {
 	c.Check(profile, testutil.FileEquals, "\n#classic"+commonPrefix+"\nprofile \"snap.samba.smbd\" (attach_disconnected) {\n\n}\n")
 }
 
+func (s *backendSuite) TestCombineSnippetsArchSufficientHardened(c *C) {
+	restore := release.MockAppArmorLevel(release.PartialAppArmor)
```

Comment:
Given that we now have a helper (downgradeConfinement) I wonder if we can put that into export_test.go and test it directly? Then we can much easier test the various combinations with e.g. a table driven test.